### ISSUE TITLE
softmax classifier seems wrong..

### ISSUE THREAD

**Issue by youngjoo-epfl** [josephpmcdonald/tensorflowtrials#5]
in tf_lstm.py at line 66, 67, code are written as follow

*******************************************************
softmax_w = tf.get_variable("softmax_w", [size, vocab_size])
softmax_b = tf.get_variable("softmax_b", [vocab_size])
*******************************************************

Should it be [size, number_of_classes]?
In sentiment analysis case, n_classes = 2

for now the code generate 10,000 output for each sample.

**Comment by jpmcd** [comment]
i believe you are incorrect. have you forked it and tried it for yourself? if you have recommendations, a pull request would be useful.
